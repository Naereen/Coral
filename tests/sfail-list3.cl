def add(x : int[]):
	acc = 0
	for i in acc:
		acc = acc + i
	return acc

type(add([1, 2, 3]))
