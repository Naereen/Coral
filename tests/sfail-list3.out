STypeError: invalid array type in for loop.